assets/js/chart_munger.js

```diff
@@ -0,0 +1,44 @@
+var munge_data = function(chart_data, chart_name) {
+  // var config = chart_data.config;
+  var data_in = chart_data;
+  var data_out = {};
+
+  data_out.labels = data_in.map(function(item) {
+      return item.name;
+  });
+
+  var munge_series = function(label) {
+    var data = {};
+    var label_lower = label.toLowerCase();
+    label_lower = label_lower.replace(' ', '_');
+
+    data.label = label;
+
+    // $.extend(data, config[label_lower]);
+
+    data.data = data_in.map(function(item) {
+      var value = chart_set.data_item_func(item, chart_name, label_lower);
+      return parseInt(value) % 1 === 0 ? parseInt(value) : 0;
```

Comment:
This was to set null (which after `parseInt` becomes NaN) to 0. But come to think of it I wrote that before the charts were working in full swing, so it might be worth checking whether this is necessary.